src/xdocs/releasenotes.xml

```diff
@@ -23,7 +23,7 @@
       <p>New:</p>
         <ul>
           <li>
-            HideUtilityClassConstructor - Add option to skip validation based on list of annotations.
+            HideUtilityClassConstructor - Option to skip validation for selected annotations.
```

Comment:
Can we just go line wrapping, no changes in wording

Reply:
Done.